### PR TITLE
v0.1.0 PR

### PR DESCRIPTION
**Pull request by yoshutch**
First pass at a release for scheduled fargate task.

This is a draft until we can get a more thorough example merged into this branch.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -1,7 +1,239 @@
...
+      security_groups = concat([aws_security_group.fargate_service_sg.id], var.security_groups)
+      subnets         = var.private_subnet_ids
+    }
+  }
+}
+
+# ==================== CloudWatch ====================
+resource "aws_cloudwatch_log_group" "container_log_group" {
+  name              = local.cloudwatch_log_group_name
+  retention_in_days = 7
```
Comment (jvisker) [inline main.tf]: Hard coded value?

Comment (yoshutch) [reply main.tf]: Done.

File: examples/simple/simple-example.tf [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  version = "~> 2.42"
+  region  = "us-west-2"
+}
+
+module "acs" {
+  source = "github.com/byu-oit/terraform-aws-acs-info?ref=v2.1.0"
+}
+
+module "scheduled_fargate" {
+  //    source = "github.com/byu-oit/terraform-aws-scheduled-fargate?ref=v0.1.0"
```
Comment (jvisker) [inline examples/simple/simple-example.tf]: Should this be the one that is commented out?

Comment (yoshutch) [reply examples/simple/simple-example.tf]: Done

File: main.tf [hunk 3]
```diff
@@ -1,7 +1,240 @@
...
+  rule      = aws_cloudwatch_event_rule.scheduled_task.name
+  arn       = var.ecs_cluster_arn == null ? aws_ecs_cluster.cluster[0].arn : var.ecs_cluster_arn
+  //  role_arn  = aws_iam_role.scheduled-task-cloudwatch.arn
+  role_arn = var.event_role_arn
+
+  ecs_target {
+    task_count          = 1
+    task_definition_arn = aws_ecs_task_definition.task_def.arn
+    launch_type         = "FARGATE"
+    platform_version    = "LATEST"
```
Comment (jvisker) [inline main.tf]: Upgrade to 1.4.0 (I believe that is the latest)

Comment (yoshutch) [reply main.tf]: Done

**Review by jvisker — APPROVED** [review]

**Pull request merged**